pkg/apis/stable/v1alpha1/gameserversort.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import "sort"
+
+// GameServerSort sorts gameservers, based on a comparator
+type GameServerSort struct {
+	// List is the GameServer list to sort
+	List []*GameServer
+	// Less is the comparator function to use as a sorting function.
+	// returns true if a < b
+	Comparator func(a, b *GameServer) bool
```

Comment:
nit: call it "Less" ?

Reply:
That was my first attempt - Go complained that I had a function and a member called `Less`, and wasn't very happy with me. `Comparator` was the best thing I could come up with.